grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package network;
+
+option go_package = "smatov/dataplane";
+
+service NetworkService {
+    rpc AddVPC(AddVPCRequest) returns (VPCResponse);
+    rpc UpdateVPC(UpdateVPCRequest) returns (VPCResponse);
+    rpc DeleteVPC(DeleteVPCRequest) returns (DeleteResponse);
+    rpc ListVPCs(ListVPCsRequest) returns (ListVPCsResponse);
+
+    rpc AddPeeringInterface(AddPIFRequest) returns (PIFResponse);
+    rpc UpdatePeeringInterface(UpdatePIFRequest) returns (PIFResponse);
+    rpc DeletePeeringInterface(DeletePIFRequest) returns (DeleteResponse);
+    rpc ListPeeringInterfaces(ListPIFsRequest) returns (ListPIFsResponse);
+
+    rpc AddPeeringPolicy(AddPeeringPolicyRequest) returns (PeeringPolicyResponse);
+    rpc UpdatePeeringPolicy(UpdatePeeringPolicyRequest) returns (PeeringPolicyResponse);
+    rpc DeletePeeringPolicy(DeletePeeringPolicyRequest) returns (DeleteResponse);
+    rpc ListPeeringPolicies(ListPeeringPoliciesRequest) returns (ListPeeringPoliciesResponse);
+}
+
+message VPC {
+    string vpc_name = 1;
+    repeated Subnet subnets = 2;
+}
+
+message Subnet {
+    string subnet_name = 1;
+    string cidr = 2;
+    uint32 vni = 3;
+}
+
+message PeeringInterface {
+    string pif_name = 1;
+    string vpc_name = 2;
+    string type = 3;
+    repeated string endpoints = 4;
+    repeated string ips = 5;
+}
+
+message PeeringPolicy {
+    string policy_name = 1;
+    repeated string peered_interfaces = 2;
```

Comment:
I don't know gRPC and here I don't know what repeated means, it seems to be 1 or more. Is there a way to tell that we need just two peered_interfaces?